src/state/apis/swapper/helpers/validateTradeQuote.ts

```diff
@@ -216,6 +216,21 @@ export const validateTradeQuote = (
             const accountId =
               portfolioAccountIdByNumberByChainId[sellAssetAccountNumber][protocolFee.asset.chainId]
             const balanceCryptoBaseUnit = portfolioAccountBalancesBaseUnit[accountId][assetId]
+
+            if (
+              firstHopSellFeeAsset?.assetId === assetId &&
+              firstHop.sellAsset.assetId === assetId
+            ) {
+              return bnOrZero(balanceCryptoBaseUnit)
+                .minus(sellAmountCryptoBaseUnit)
+                .minus(protocolFee.amountCryptoBaseUnit)
+                .lt(0)
+            }
+
+            console.log(
+              'not sell asset',
+              bnOrZero(balanceCryptoBaseUnit).lt(protocolFee.amountCryptoBaseUnit),
+            )
```

Comment:
rm?

Reply:
fml, removed in [6840b5c](https://github.com/shapeshift/web/pull/8267/commits/6840b5c3ff9eefd79239117bbd0c17cdb1c4bebf)